src/spaces/openstreetmap.jl

```diff
@@ -662,3 +662,69 @@ function nearby_positions(
     end
     neighborfn(model.space.m.g, position)
 end
+
+
+
+#returns sparse matrix with travel times between connected nodes
+function get_travel_times(m::MapData, class_speeds::Dict{Int,Float64} = OpenStreetMapX.SPEED_ROADS_URBAN)::SparseMatrixCSC{Float64,Int64}
```

Comment:
There is no gain here in having type anotations for both `class_speeds` or the return value of the function (Sparse matrix)